common/lib/src/realm_types.dart

```diff
@@ -316,17 +314,15 @@ enum SyncConnectionErrorCode {
   switchToFlxSync(113),
 
   /// Connected with wrong wire protocol - should switch to PBS
-  switchToPbs(114);
+  switchToPbs(114),
+
+  /// Unknown error code
```

Comment:
```suggestion
  /// Unknown Sync connection error code
```

Reply:
instead of generic api doc, we could mention the category here. like the suggestion above. I suggest we do this for each sync error type